codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -817,6 +822,15 @@ private void notifyGoingAway() {
             }
         }
 
+        private void closeStreamsGreaterThanLastKnownStream() {
+            Http2Stream[] streams = new Http2Stream[numActiveStreams()];
+            for (Http2Stream stream : activeStreams().toArray(streams)) {
+                if (stream.id() > lastKnownStream()) {
```

Comment:
@buchgr I think you need to make sure that the stream was created by `this` endpoint :)